docker-compose-tests.yml

```diff
@@ -10,7 +10,7 @@ services:
     volumes:
       - gp-finder-tests-node_modules:/code/node_modules
     environment:
-      NODE_ENV: test
+      NODE_ENV: ${NODE_ENV-test}
```

Comment:
Does the variable want setting when the value has is either unset or empty or just when it is unset?
See docs for more info: 
https://docs.docker.com/compose/compose-file/compose-file-v2/#variable-substitution